pom.xml

```diff
@@ -38,34 +39,36 @@
   </licenses>
 
   <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <jdk.version>1.8</jdk.version>
+
     <buildnumber.maven.plugin.version>1.4</buildnumber.maven.plugin.version>
     <coveralls.maven.plugin.version>4.3.0</coveralls.maven.plugin.version>
-    <failsafe.maven.plugin.version>2.22.0</failsafe.maven.plugin.version>
     <eg.oss.plugin.config.version>1.1.0</eg.oss.plugin.config.version>
+    <failsafe.maven.plugin.version>2.22.0</failsafe.maven.plugin.version>
```

Comment:
This and all the other changes in this section are just alphabetic re-ordering of what we already had, I didn't change anything else.